docs/source/develop.rst

```diff
@@ -68,9 +68,12 @@ Development Environment
 Download code
 ~~~~~~~~~~~~~
 
-Clone the main Dask git repository (or whatever repository you're working on)::
+Make a fork of the main `Dask repository <https://github.com/dask/dask>`_ and
+clone the fork::
 
-   git clone git@github.com:dask/dask.git
+   git clone https://github.com/<your-github-username>/dask
```

Comment:
I would actually recommend to clone the main repo and add your fork as an additional remote. Otherwise you won't get updates from master in your clone.

Reply:
Thanks for the review @xhochy, that's a good point. However since we don't go into much detail about git workflows here, and instead link to the pandas contributing guide which recommends cloning the fork and adding the main repo as an `upstream` remote, I have a slight preference for staying with cloning the fork. I think this consistency could help contributors who are new to git or GitHub.

Reply:
Agreed. I personally add the main branch as `upstream`, and keep `origin` as my fork. Updating from master is then just `git pull upstream master`.